### ISSUE TITLE
How to specify eslint config

### ISSUE THREAD

**Issue by keenahn** [the-simian/es6-plato#36]
I'm getting a lot of lint errors in the report, and it's because plato is not picking up my `.eslintrc` any thoughts?

**Comment by the-simian** [comment]
offhand not sure, are you using es6-plato from the command line, or in the context of another build tool?

**Comment by texels** [comment]
I am getting the same issues running from the command line: es6-plato -r -d report src

**Comment by the-simian** [comment]
thanks @texels !, can you lmk your eslint version just so I can test repro :)

**Comment by pats** [comment]
me too
I'm getting a lot of issues 
especially
 Cannot find module 'eslint-plugin-react'

**Comment by the-simian** [comment]
Hey everyone I am sorry you're having problems, can you post actual debugging information?

**Comment by gknapp** [comment]
I've installed `es6-plato` globally on my CI server.

```
> es6-plato -v
1.0.16
```

`> es6-plato -r -x "(styles|assets|.+\.(html|bak|json))" -d report app`

```
Error reading file app/redux/reducers/data/stored-procedures.js:  Error: Failed to load plugin react: Cannot find module 'eslint-plugin-react'
Error: Failed to load plugin react: Cannot find module 'eslint-plugin-react'
    at Function.Module._resolveFilename (module.js:555:15)
    at Function.resolve (internal/module.js:18:19)
    at Plugins.load (C:\Users\gregk\AppData\Roaming\npm\node_modules\es6-plato\node_modules\eslint\lib\config\plugins.js:106:29)
    at Array.forEach (<anonymous>)
    at Plugins.loadAll (C:\Users\gregk\AppData\Roaming\npm\node_modules\es6-plato\node_modules\eslint\lib\config\plugins.js:146:21)
    at loadFromDisk (C:\Users\gregk\AppData\Roaming\npm\node_modules\es6-plato\node_modules\eslint\lib\config\config-file.js:492:35)
    at Object.load (C:\Users\gregk\AppData\Roaming\npm\node_modules\es6-plato\node_modules\eslint\lib\config\config-file.js:550:20)
    at Config.getLocalConfigHierarchy (C:\Users\gregk\AppData\Roaming\npm\node_modules\es6-plato\node_modules\eslint\lib\config.js:228:44)
    at Config.getConfigHierarchy (C:\Users\gregk\AppData\Roaming\npm\node_modules\es6-plato\node_modules\eslint\lib\config.js:180:43)
    at Config.getConfigVector (C:\Users\gregk\AppData\Roaming\npm\node_modules\es6-plato\node_modules\eslint\lib\config.js:287:21)
```

Lots of lines of the above, ending with:

```
Error: Failed to load plugin react: Cannot find module 'eslint-plugin-react'
    at Function.Module._resolveFilename (module.js:555:15)
    at Function.resolve (internal/module.js:18:19)
    at Plugins.load (C:\Users\gregk\AppData\Roaming\npm\node_modules\es6-plato\node_modules\eslint\lib\config\plugins.js:106:29)
    at Array.forEach (<anonymous>)
    at Plugins.loadAll (C:\Users\gregk\AppData\Roaming\npm\node_modules\es6-plato\node_modules\eslint\lib\config\plugins.js:146:21)
    at loadFromDisk (C:\Users\gregk\AppData\Roaming\npm\node_modules\es6-plato\node_modules\eslint\lib\config\config-file.js:492:35)
    at Object.load (C:\Users\gregk\AppData\Roaming\npm\node_modules\es6-plato\node_modules\eslint\lib\config\config-file.js:550:20)
    at Config.getLocalConfigHierarchy (C:\Users\gregk\AppData\Roaming\npm\node_modules\es6-plato\node_modules\eslint\lib\config.js:228:44)
    at Config.getConfigHierarchy (C:\Users\gregk\AppData\Roaming\npm\node_modules\es6-plato\node_modules\eslint\lib\config.js:180:43)
    at Config.getConfigVector (C:\Users\gregk\AppData\Roaming\npm\node_modules\es6-plato\node_modules\eslint\lib\config.js:287:21)fs.js:663
  return binding.open(pathModule.toNamespacedPath(path),
                 ^

Error: ENOENT: no such file or directory, open 'c:\Program Files (x86)\Jenkins\workspace\Manager\report\report.history.json'
    at Object.fs.openSync (fs.js:663:18)
    at Object.fs.writeFileSync (fs.js:1322:33)
    at writeFile (C:\Users\gregk\AppData\Roaming\npm\node_modules\es6-plato\lib\plato.js:292:5)
    at writeReport (C:\Users\gregk\AppData\Roaming\npm\node_modules\es6-plato\lib\plato.js:298:2)
    at updateHistoricalOverview (C:\Users\gregk\AppData\Roaming\npm\node_modules\es6-plato\lib\plato.js:278:2)
    at C:\Users\gregk\AppData\Roaming\npm\node_modules\es6-plato\lib\plato.js:194:6
    at mkdir.mkdirs.err (C:\Users\gregk\AppData\Roaming\npm\node_modules\es6-plato\node_modules\fs-extra\lib\copy\copy.js:47:25)
    at xfs.stat (C:\Users\gregk\AppData\Roaming\npm\node_modules\es6-plato\node_modules\fs-extra\lib\mkdirs\mkdirs.js:55:43)
    at C:\Users\gregk\AppData\Roaming\npm\node_modules\es6-plato\node_modules\graceful-fs\polyfills.js:284:29
    at FSReqWrap.oncomplete (fs.js:166:21)
```

(I am running this manually from my CI server in the Jenkins workspace to get the command right before I add it to the pipeline).

Do the end lines look like a permissions issue?

**Comment by gknapp** [comment]
I installed this as a `devDependency` in my development workspace, it ran without issue:

```
> npm i -D es6-plato
+ es6-plato@1.0.16
added 16 packages in 19.59s

> node_modules\.bin\es6-plato.cmd -r -x "(styles|assets|.+\.(html|bak|json))" -d report app
Done!
```

**Comment by the-simian** [comment]
hey @gknapp thanks for the feedback (and the stack trace!!) , I think that permissions could be related, but its good to see the 'problem module' is 'eslint-plugin-react'

**Comment by the-simian** [comment]
@keenahn , @gknapp , @pats , @texels I have a released a new version, I added the offending plugin as a dependency directly. (it had better find it now, right?) I tested installing and it worked for me with both yarn and also standard npm.

I want you to try this version and see how it goes.

I'm closing the issue, but if the problem is still there, just post the stack trace like @gknapp did and I'll dig deeper.

Thank you all for reporting the issue <3
